xdsl/dialects/mpi.py

```diff
@@ -317,7 +350,7 @@ def get(buffer: SSAValue | Operation, count: SSAValue | Operation,
 
 
 @irdl_op_definition
-class IRecv(MPIBaseOp):
+class Irecv(MPIBaseOp):
```

Comment:
I slightly prefer the two capitals at the start, but I don't mind too much.

Reply:
Same

Reply:
It's to be "consistent" with MPI naming. It's called "Irecv" there, sadly :(